src/config/dynamic-theme-fixes.config

```diff
@@ -14653,6 +14653,15 @@ INVERT
 
 ================================
 
+photos.google.com
+
+CSS
+textarea {
+    background-color: #00000000 !important;
```

Comment:
Could you try if:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```
or
```suggestion
    background-color: ${white} !important;
```
works as well?

Reply:
Neither of them works. `${white}` resolves to `black`, which doesn't match the background.

Weirdly enough, even `var(--darkreader-neutral-background)` doesn't work; it's darker than it should be. I checked the style for the original webpage, and the background color is white; So I can't figure out why this does not work.

Just out of curiosity, is this a matter of coding style and readability? Or is there a performance penalty with using transparency? Or is it something else altogether?

Reply:
> Just out of curiosity, is this a matter of coding style and readability? Or is there a performance penalty with using transparency? Or is it something else altogether?

Overall transparent is a hack and not an real fix.

Reply:
```suggestion
    background-color: transparent !important;
```

Reply:
Alright. Thanks!

Reply:
For now transparent should be good, but more or less because Google has some funky CSS that is causing this.